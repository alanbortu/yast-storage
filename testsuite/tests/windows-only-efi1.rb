# encoding: utf-8

# testedfiles: helper.rb

module Yast

  class TestClient < Client

    def main

      Yast.include self, "setup-system.rb"

      setup_system("windows-only-efi")

      def setup2()
        ProductFeatures.SetBooleanFeature("partitioning", "try_separate_home", true)
        ProductFeatures.SetBooleanFeature("partitioning", "proposal_lvm", false)
        ProductFeatures.SetStringFeature("partitioning", "root_max_size", "20 GB")
        ProductFeatures.SetStringFeature("partitioning", "root_base_size", "15 GB")
        ProductFeatures.SetBooleanFeature("partitioning", "proposal_snapshots", false)
      end

      def setup3()
      end

      Yast.include self, "helper.rb"

    end

  end

end

Yast::TestClient.new.main
