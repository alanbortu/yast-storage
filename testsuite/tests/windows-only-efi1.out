Dump	Proposal:
Dump	Shrink Windows partition /dev/sda2 to 7.78 GiB [destructive]
Dump	Create swap volume /dev/sda3 (517.72 MiB)
Dump	Create root volume /dev/sda4 (7.62 GiB) with btrfs
Dump	Create subvolume boot/grub2/i386-pc on device /dev/sda4
Dump	Create subvolume boot/grub2/x86_64-efi on device /dev/sda4
Dump	Create subvolume opt on device /dev/sda4
Dump	Create subvolume srv on device /dev/sda4
Dump	Create subvolume tmp on device /dev/sda4
Dump	Create subvolume usr/local on device /dev/sda4
Dump	Create subvolume var/crash on device /dev/sda4
Dump	Create subvolume var/log on device /dev/sda4
Dump	Create subvolume var/opt on device /dev/sda4
Dump	Create subvolume var/spool on device /dev/sda4
Dump	Create subvolume var/tmp on device /dev/sda4
Dump	Set mount point of /dev/sda1 to /boot/efi
